tests/FakeItEasy.Specs/XBehaveExtensions.cs

```diff
@@ -0,0 +1,70 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using Creation;
+    using Xbehave;
+    using Xbehave.Sdk;
+
+    public static class XBehaveExtensions

```

Comment:
My initial reasoning had been along the same lines as @thomaslevesque, but in some sense, that's the case for other extensions that we write: they're really for _our_ use, regardless of the type they "extend". E.g., `TypeExtensions.GetDefaultValue`.
Unless @thomaslevesque objects, I am inclined to rename the class `StringExtensions`, following the other examples. The system is easy to remember (my example today notwithstanding) and apply, with no soul-searching required.

If nothing else, switching to a more sophisticated system as @thomaslevesque hints at could be done as part of a separate change…